backend/src/app/services/parcelDescriptions/parcelDescriptions.service.ts

```diff
@@ -461,6 +461,49 @@ export class ParcelDescriptionsService {
     siteId: string,
     parcelDescriptions: ParcelDescriptionInputDTO[],
   ) {
-    // TODO: Implementation to come in another pull request
+    this.sitesLogger.log(
+      'parcelDescriptionService.deleteParcelDescriptionsForSite() start',
+    );
+    this.sitesLogger.debug(
+      'parcelDescriptionService.deleteParcelDescriptionsForSite() start',
+    );
+
+    const subdivIds = parcelDescriptions.map((parcelDescription) => {
+      return parcelDescription.id;
+    });
+    let siteSubdivisions = await this.siteSubdivisionsRepository.findBy({
+      subdivId: In(subdivIds),
+      siteId: siteId,
+    });
+    let subdivisions = await this.subdivisionsRepository.findBy({
+      id: In(subdivIds),
+    });
+
+    try {
+      await this.subdivisionsRepository.remove(subdivisions);
```

Comment:
All database operations should be executed through transaction manager as we work with multiple entities on saving.  Please rewrite the code to use transaction manager instead of accessing repository directly  for example please refer to implementation in land.history.service.ts

Reply:
Should this code be called outside the transaction block in the sites service? If so I will raise another PR to refactor the sites service so that saving site details does not include parcel descriptions within a transaction.